src/tinypeel/Peeling/PeelingIO.py

```diff
@@ -75,10 +75,10 @@ def writeOutParamaters(peelingInfo):
     np.savetxt(args.out + ".alt_allele_prob.txt", peelingInfo.alt_allele_prob, fmt="%f")
 
 
-def writeGenotypes(pedigree, genoProbFunc):
+def writeGenotypes(pedigree, genoProbFunc, isSexChrom):
```

Comment:
@AprilYUZhang I understand what you are doing here, but this is a departure from how you handled the option name changes before. Is there a particular reason that you are now picking a different strategy. Doing that makes code even more non-uniform and if you do this a couple of times, the code style becomes impossible to follow.

Reply:
@AprilYUZhang OK I propose the following - when you encounter such inconsistencies, open an issue so we can later fix them. By finding these problems, you have already spent quite a bit of time understanding the logic so if you open an issue and explain this in the issue content, then that effort is not lost and it easier to pick up at a later stage when you get back to the issue. OK?